pkg/controller/clusterdeployment/clusterdeployment_controller_test.go

```diff
@@ -1843,3 +1867,39 @@ func getProvisions(c client.Client) []*hivev1.ClusterProvision {
 	}
 	return provisions
 }
+
+func createSyncSetInstanceObj(syncCondType hivev1.SyncConditionType) *hivev1.SyncSetInstance {
+	ssi := &hivev1.SyncSetInstance{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      testSyncsetInstanceName,
+			Namespace: constants.HiveNamespace,
+		},
+	}
+	ssi.Spec.ClusterDeployment.Name = testClusterName
```

Comment:
The name of the cluster should be `testName`.

Reply:
Are you sure about this as I can see the cluster deployment name is ` testClusterName`
```
func testClusterDeployment() *hivev1.ClusterDeployment {
	cd := testEmptyClusterDeployment()

	cd.Spec = hivev1.ClusterDeploymentSpec{
		ClusterName: testClusterName,
```